### ISSUE TITLE
URI check is too green

### ISSUE THREAD

**Issue by matentzn** [OBOFoundry/OBO-Dashboard#76]
Look into it in the future, I doubt no ontologies has broken URIs for terms.

**Comment by anitacaron** [comment]
It would be good to add the Ontology URI to the report if it's wrong.

For example, for [agro](https://dashboard.obofoundry.org/dashboard/agro/dashboard.html)

![Screenshot 2023-12-20 at 12 23 39](https://github.com/OBOFoundry/OBO-Dashboard/assets/2208124/959f12ba-e5be-4dc8-bf91-65dd721fa092)

But the report is empty because no URI is incorrect, only the Ontology URI

![Screenshot 2023-12-20 at 12 23 58](https://github.com/OBOFoundry/OBO-Dashboard/assets/2208124/56aa7574-a333-4898-98d8-ee0d5561526b)

**Comment by matentzn** [comment]
I agree - since this probably does not generalise too well beyond OBO, maybe adding it to ROBOT is overkill (https://robot.obolibrary.org/report_queries/) but you could add a custom sparql query like you did on your custom OBO dashboard to cover this case!

**Comment by anitacaron** [comment]
Checking the NOR Dashboard, two ontologies with invalid URIs passed the URI check. In the first case, RTO does have an almost empty base file, so there is nothing to report, but it should somehow have a message when no class was found. The same happens to the MODAL ontology, which uses the OWL/XML serialisation. Need to give more warning messages in these cases.

**Comment by matentzn** [comment]
Thanks for following up on this. Yes I fully agree..
